docs/marketplace/github-marketplace/enterprise-store/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Setting Up Appcircle Enterprise Store Action
+sidebar_label: Enterprise App Store
+description: Enhance powerful action to publish your builds to appcircle app store
+tags:
+  [
+    overview,
+    concepts,
+    app store,
+    internal testing,
+    beta testing,
+    binary distribution,
+    ipa distribution,
+    apk distribution,
+  ]
+sidebar_position: 2
+---
+
+The Appcircle Enterprise Store action allows users to publish their apps to appcircle app store
```

Comment:
The Appcircle Enterprise Store action allows users to publish their apps to Appcircle App Store.